### PR TITLE
Auto-scrolling

### PR DESCRIPTION
**Pull request by nabalone**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/EthnoLib/43)

### CODE REVIEW

File: components/language-chooser/react/language-chooser-react-mui/src/LanguageChooser.tsx [hunk 1]
```diff
@@ -275,12 +271,21 @@ export const LanguageChooser: React.FunctionComponent<ILanguageChooserProps> = (
...
                   ref={
                     isSelectedLanguageCard ? selectedLanguageCardRef : undefined
                   }
+                  onMount={() => {
+                    if (
+                      props.initialSelectionLanguageTag &&
+                      isSelectedLanguageCard
+                    ) {
+                      // This should only run once
+                      setInitialScrollingNeeded(true);
```
Comment (nabalone) [inline components/language-chooser/react/language-chooser-react-mui/src/LanguageChooser.tsx:286]: I'm don't understand why this only ever runs once, even if you change the search string and then select a card that wasn't there before. But it works! Running only once is the desired behavior

**Comment by nabalone** [comment]
I have two implementations (the two commits). Both work. I don't understand why either is working. Please take a look at the diff: https://github.com/sillsdev/EthnoLib/pull/43/commits/3880bb802f2944f1a503633e2375492a711f50e3?diff=split&w=0

**Review by andrew-polk — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/EthnoLib/43)* status: all files reviewed, 2 unresolved discussions

**Review by andrew-polk — APPROVED** [review]
Update: we discussed, decided we understand, she commented, and we're good.

> *[Reviewable](https://reviewable.io/reviews/sillsdev/EthnoLib/43#-:-ODhWsAV2Gj8ffgmLDfa:bk1nr1)* status: all files reviewed, 2 unresolved discussions

**Review by andrew-polk — APPROVED** [review]
> Dismissed @nabalone from 2 discussions.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/EthnoLib/43)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @nabalone)

**Pull request merged**